draft-ietf-acme-acme.md

```diff
@@ -1972,6 +1997,11 @@ domain by verifying that the resource was provisioned as expected.
 5. Verify that key authorization provided by the server matches the token for
    this challenge and the client's account key.
 
+The server MAY follow redirects when dereferencing the URI.
+
+It is RECOMMENDED that the server dereference the URI multiple times from
+various network perspectives, in order to make MitM attacks harder.
```

Comment:
Same comment here.  New normative language, and it implies that server's can't just take down the verification thing after they get a query, but rather keep it up for some indeterminate period of time.